DOC-1765-outdegree-stmt-level (#105)
tigergraph/gsql-docs: modules/querying/pages/func/vertex-methods.adoc

Returns the number of outgoing or undirected edges connected to the vertex. If edge types are provided, it will only return the number of edges of the specified types.

[NOTE]
====

> (for 3.9.2+) `outdegree()` can also be using a vertex input parameter.

Author: ```suggestion
(for 3.9.2+) `outdegree()` is also available for vertex input parameters.
```
